SwiftGit2Tests/RepositorySpec.swift

```diff
@@ -641,8 +641,63 @@ class RepositorySpec: QuickSpec {
 				expect(commitMessages).to(equal(expectedMessages))
 			}
 		}
+
+		describe("Repository.getRepositoryStatus") {
+			it("Should return accurate status") {
+				// Repo with no status
+				let repo = Fixtures.mantleRepository
+				let branch = repo.localBranch(named: "master").value!
+				expect(repo.checkout(branch, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let status = repo.status()
+
+				expect(status.value?.count).to(equal(0))
+
+				let repoWithStatus = Fixtures.sharedInstance.repository(named: "repository-with-status")
+				let branchWithStatus = repoWithStatus.localBranch(named: "master").value!
+				expect(repoWithStatus.checkout(branchWithStatus, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let statusWithStatus = repoWithStatus.status()
+
+				expect(statusWithStatus.value?.count).to(equal(5))
+			}
+
+			it("Should have accurate delta information") {
+				let repo = Fixtures.mantleRepository
+				let branch = repo.localBranch(named: "master").value!
+				expect(repo.checkout(branch, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let head = repo.HEAD().value!
+				let commit = repo.object(head.oid).value! as! Commit
+				let objects = repo.diff(for: commit)
```

Comment:
This test (and the following ones) are testing `diff`, so they should be in a separate `describe`.